src/main/java/uk/gov/hmcts/darts/retention/exception/RetentionApiError.java

```diff
@@ -21,19 +21,19 @@ public enum RetentionApiError implements DartsApiError {
         RetentionTitleErrors.RETENTION_DATE_TOO_EARLY.toString()
     ), INVALID_REQUEST(
         RetentionErrorCode.INVALID_REQUEST.getValue(),
-        HttpStatus.BAD_REQUEST,
+        HttpStatus.UNPROCESSABLE_ENTITY,
         RetentionTitleErrors.INVALID_REQUEST.toString()
     ), CASE_NOT_FOUND(
         RetentionErrorCode.CASE_NOT_FOUND.getValue(),
-        HttpStatus.BAD_REQUEST,
+        HttpStatus.UNPROCESSABLE_ENTITY,
```

Comment:
If a case isn't found, don't we want to know about it in the logs?

Reply:
This will be fine on the FE